src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaJsonWriter.Async.cs

```diff
@@ -0,0 +1,1722 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="EdmModelCsdlSchemaJsonWriter.Async.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+#if NETSTANDARD2_0
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Microsoft.OData.Edm.Csdl.CsdlSemantics;
+using Microsoft.OData.Edm.Csdl.Parsing.Ast;
+using Microsoft.OData.Edm.Vocabularies;
+using Microsoft.OData.Edm.Vocabularies.V1;
+
+namespace Microsoft.OData.Edm.Csdl.Serialization
+{
+    /// <summary>
+    /// OData Common Schema Definition Language (CSDL) JSON writer
+    /// </summary>
+    internal partial class EdmModelCsdlSchemaJsonWriter : EdmModelCsdlSchemaWriter

```

Comment:
Across this file: Why are we calling the synchronous methods for the `IJsonWriter` instance instead of the asynchronous equivalent?

Reply:
`Utf8JsonWriter` does not contain a definition for `Async` methods apart from `FlushAsync` and `DisposeAsync`
`jsonWriter` here is an instance of `Utf8JsonWriter`

[utf8jsonwriter](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonwriter?view=net-8.0)